docs/EnvironmentVariables.md

```diff
@@ -1,7 +1,10 @@
 # Environment variables
 
```

Comment:
I suggest that the following is inserted:

```
## Variable used by both RPCAPI daemon and Test Agent daemon
```

(We should be consistent if it should be "Test agent", "TestAgent" or "Test Agent". It differs in different documents.)

Reply:
done